utils/build-icons/lib/config.js

```diff
@@ -105,7 +105,7 @@ module.exports = {
         removeDimensions: true
       },
       {
-        removeAttrs: { attrs: '(stroke|fill)' }
+        removeAttrs: false
       }
     ]
   }
```

Comment:
Why are we removing so much of the optimization? I know @aselbie mentioned he's seen it fail or mess with the intended vectors (definitely curious to hear a little more about that).

In the last SVG icon set I helped implement I believe we tuned it pretty similarly to how this had been configured and only whitelisted a couple attributes (`class` and `currentColor`?) used in targeting individual SVG nodes with CSS. This worked great.

That being said I understand we may want to be a little cautious with optimizations that actually affect `path` data like `convertPathData`, `convertTransform`, `mergePaths`, and `convertShapeToPath`.

I'd be willing to do some more research/testing and follow up with a PR that re-enables some based on more information – but I still think we can trust more/most of these ;)

Reply:
I don't see much value if we are sure to only send one copy of the used icon down per page. If we determine they are bigger we can re-enable some or we can get design to run something manually before committing it to the repo.

Reply:
Alright. So looking at this again, my experience with SVGO is mostly through https://jakearchibald.github.io/svgomg/. By default it has almost every setting on. If there are particular settings that you all are familiar with that have measurable benefits and limited downsides I'm completely cool with that. My experience with this particular tool is that it has broken a few icons and I've had to play with the settings (sometimes manually) until it didn't.

In general, though, me feeling is that we'll be adding icons at an asymptotically decreasing rate. I'm not sure that the effort/risk involved in automating (or adding it to the build pipeline in general) this optimization is worth the potential downsides.

Reply:
> asymptotically decreasing rate

Reply:
I tend to agree with @aselbie and also, we can change the optimization rules later in a follow up if needed.

Reply:
> I'd be willing to do some more research/testing and follow up with a PR that re-enables some based on more information
= follow up